docs/api/Events.md

```diff
@@ -222,6 +222,28 @@ res = emitter.emit('event3'); // res == false
 
 ```
 
+### emitter.setMaxListeners(num)
+* `num` {number} The max listener limit, default 10.
+* Returns `emitter` {events.EventEmitter}.
+
+Set the max listener limit, must be number and > 0.
+
+### emitter.getMaxListeners()
+* Returns {number} The max listener limit.
+
+Get the max listener limit, must be number.
+
+### emitter.setDefaultMaxListeners(num)
```

Comment:
I'm wondering if `setDefaultMaxListeners` and `getDefaultMaxListeners` shall be an instance method rather than a static method, yet since EventEmitter.defaultMaxListeners could be defined as an accessor, both `setDefaultMaxListeners` and `getDefaultMaxListeners` are redundant in my opinion.

Reply:
Exactly, I will define an accessor. Thanks!